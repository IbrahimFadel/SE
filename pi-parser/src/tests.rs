#[cfg(test)]
#[test]
fn expr() {
	//   use pi_ast::{Expr, BinOp, IntLit};

	// let tests = [
	//   ("1 + 1", (
	//     // vec![Expr::BinOp(BinOp::new(IntLit::new(sign_span, val_span, signed, bits, val), op, y))]
	//     vec![Expr::BinOp(BinOp{})],
	//   ))
	// ]
}
