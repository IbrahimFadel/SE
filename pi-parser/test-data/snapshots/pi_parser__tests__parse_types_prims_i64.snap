---
source: pi-parser/src/tests.rs
expression: "&expr"
---
node:
  PrimitiveType: I64
span:
  range:
    start: 0
    end: 3
  file_id: 0

